tests/test_setutils.py

```diff
@@ -66,3 +73,59 @@ def big_popper():
     print()
     print('popped %s items in %s seconds' % (start_size - end_size,
                                              end_time - start_time))
+
+
+def test_complement_set():
```

Comment:
I like how concise this test is, especially in the initial "just getting it working" phase, but this should be split into separate tests, some of which are created using [`pytest.mark.parametrize`](https://docs.pytest.org/en/latest/parametrize.html) and the 
[operator](https://docs.python.org/3/library/operator.html) module, with the 2.6 skips skipped with the [`pytest.mark.skipif`](https://docs.pytest.org/en/latest/skipping.html#skipif) marker. 
Probably want to wrap it in a `class TestComplementSet`, so you don't feel like you have to include "complement_set" in every test name.
That said, it's not immediately obvious how the tests should be grouped for parameterization.
I would probably start by moving things around until I could get a number of assertions that could be generalized with parameters "operation, expected_result".
Note that when you extract fixtures, you cannot use them explicitly in `pytest.mark.parametrize` without resorting to a 3rd party pytest plugin, so that will be tricky.